src/esm.ts

```diff
@@ -93,6 +94,7 @@ export function registerAndCreateEsmHooks(opts?: RegisterOptions) {
 
 export function createEsmHooks(tsNodeService: Service) {
   tsNodeService.enableExperimentalEsmLoaderInterop();
+  const mapPath = createPathMapper(tsNodeService.config.options);
```

Comment:
We can move creation of this into `create()` and store it on the service as `tsNodeService.pathMapper` since it'll be used for CommonJS, too.

Reply:
https://github.com/TypeStrong/ts-node/pull/1585/commits/2fcb9821593fd9235f45585ed0c95b1943a9c3de